### PR TITLE
Fix net overflowing on large damage values

### PR DESCRIPTION
**Pull request by wrefgtzweve**
error:
```
[cfc_pain_sounds] Trying to send an overflowed net message!
  1. fn - addons/cfc_pain_sounds/lua/cfc_pain_sounds/sv_pain_relay.lua:60
   2. unknown - lua/ulib/shared/hook.lua:109
    3. FireBullets - [C]:-1
     4. unknown - lua/weapons/weapon_ak47_admin.lua:156
```

net msg size before:
```
1    5    40
2    65536    524288
```
as can be seen, when damage is high the byte amount is too much for net and it errors, in this case it's the css admin ak doing 250k dmg per shot

after:
```
1	5	40
2	7	52
```

image

### CODE REVIEW

**Review by overlordbleck — DISMISSED** [review]
I'm literally puking at the sight of this

File: moon/cfc_pain_sounds/sv_pain_relay.moon [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import Recipientfilter, IsValid, CurTime from _G
+math_min = math.min
```
Comment (sarahsturgeon) [inline moon/cfc_pain_sounds/sv_pain_relay.moon]:
```suggestion
import min from math
```

I think `min` is descriptive enough to use without the library prefix, but I won't hold up the PR for it

File: moon/cfc_pain_sounds/sv_pain_relay.moon [hunk 2]
```diff
@@ -24,7 +25,7 @@ hook.Add "PostEntityTakeDamage", "CFC_PainSounds", (victim, dmg, took) ->
 
     net.Start "CFC_PainSounds_TookDamage", unreliable
     net.WriteEntity victim
-    net.WriteUInt 7, damageAmount
+    net.WriteUInt 7, math_min 1000, damageAmount -- Avoids net overflow when dmg is incredibly high
```
Comment (sarahsturgeon) [inline moon/cfc_pain_sounds/sv_pain_relay.moon]: nit: I'd personally leave the comment out

**Review by sarahsturgeon — DISMISSED** [review]

**Comment by sarahsturgeon** [comment]
Another option is redefining it earlier in the function, I think that'd be the best of both worlds. Descriptive, keeps the net area cleaner

**Review by sarahsturgeon — APPROVED** [review]

**Pull request merged**